### ISSUE TITLE
Two levels of Lower Initiative debuff

### ISSUE THREAD

**Issue by Positiff88** [VladimirMakeev/D2ModdingToolset#217]
Please add a second level of Lower Initiative debuff (similarly to Lower Damage debuff). If it is possible, it will be nice to have BATLOWERI2 field for it in GVars.dbf and ability to set LEVEL field to 2 in Gattacks.dbf for Lower Initiative attack (CLASS = 11).

**Comment by mortling** [comment]
I think there is no need to follow vanilla levelled debuff pattern. Two levels are too few for modders (whose requirements are demanding). When it comes to this specific attack class and lowering of damage as well, it would be better to allow to set any value within [1;100] range for each separate attack entry in Gattacks.dbf. Thus I see no need to put effort in improvements that don’t result into flexible solutions.

**Comment by Positiff88** [comment]
I agree, it is better to set effect percentage instead of levels. But it can be technically hard to change original behavior in such significant way. As another, possibly easier to implement, option the amount of effect levels can be increased to 10 or 20 for Lower Initiative, Lower Damage, Boost Damage effects with fixed amount of bonus for every level (which can be specified in  existing BATLOWERI1, BATLOWERD1, BATBOOSTD1 fields and multiplied on effect level).
